web/modules/custom/hoeringsportal_audit_log/README.md

```diff
@@ -117,11 +123,17 @@ types:
   user:
     user:
       -
-        entity__dot__user__dot__canonical: 0
-        entity__dot__user__dot__edit_form: 0
+        entity__dot__user__dot__canonical: entity__dot__user__dot__canonical # Enabled
+        entity__dot__user__dot__edit_form: entity__dot__user__dot__edit_form # Enabled
         entity__dot__user__dot__cancel_form: 0
         entity__dot__user__dot__collection: 0
         entity__dot__user__dot__display: 0
+routes_to_log:
+  url_pattern:
+    - '/^\/admin\/content\?title=&type=All&status=1$/'
```

Comment:
Consider telling that/if this is a list of PHP regex patterns (cf. https://www.php.net/manual/en/reference.pcre.pattern.syntax.php).